src/persist/src/sqlite.rs

```diff
@@ -208,6 +182,77 @@ impl Consensus for SqliteConsensus {
             Ok(Err(current))
         }
     }
+
+    async fn scan(
+        &self,
+        _deadline: Instant,
+        key: &str,
+        from: Option<SeqNo>,
+    ) -> Result<Vec<VersionedData>, ExternalError> {
+        let conn = self.conn.lock().await;
+        let mut stmt = conn.prepare_cached(
+            "SELECT sequence_number, data FROM consensus
+                 WHERE shard = $shard AND sequence_number >= $from
+                 ORDER BY sequence_number DESC",
```

Comment:
hmm I don't have a strong justification for this, but my instinct is that I'd expect them to be ascending (in fact, didn't even notice what order they were in the test until I saw this and went back and checked). we should explictly make this call one way or the other and add it to the rustdoc on the method

Reply:
done and done.